### ISSUE TITLE
2D Platformer - one-way platform malfunctions

### ISSUE THREAD

**Issue by agameraaron** [godotengine/godot#2038]
OSX only? Using version 1.1 stable.
Video shows bug in detail. It looks like he explodes then is deleted...? As you can see, I can still control the camera, but it is only once I "move" the now non-existent main character to all the way to the edges: https://youtu.be/Ngi-YJWPDsc

Another issue already listed is similar in that it is triggered the same, but seems to have different results in Android: https://github.com/okamstudio/godot/issues/1988

**Comment by akien-mga** [comment]
Duplicate of #2345.
